src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -382,15 +382,14 @@ public void Timeout_TooShort_AllPendingOperationsCanceled()
 
         [Fact]
         [OuterLoop]
-        [ActiveIssue(12778)]
         public void Timeout_SetTo60AndGetResponseFromServerWhichTakes40_Success()
         {
             // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
             const string SlowServer = "http://httpbin.org/drip?numbytes=1&duration=1&delay=40&code=200";
             
             using (var client = new HttpClient())
             {
-                client.Timeout = TimeSpan.FromSeconds(60);
+                client.Timeout = TimeSpan.FromMinutes(5);
```

Comment:
> The point of this test is to verify that the operation of HttpClient Timeout feature is working as expected. And we choose 60 seconds because it is the non-default value of that property.

Can we just have this go against a loopback server?  We can then make the loopback server never respond, and choose a very short timeout, like 100ms, so the test will be both fast and reliable.